### PR TITLE
Update templates to be lint-clean under new default config

### PR DESCRIPTION
**Pull request by rachelgshaffer**
This PR contains some small changes to the templates so that creating projects will be lint-clean under the default ESLint configuration in VS.

### CODE REVIEW

File: Nodejs/Product/Nodejs/ProjectTemplates/AzureNodejsWorker/worker.js [hunk 1]
```diff
@@ -4,6 +4,6 @@
  */
 'use strict';
 
-while (true) {
```
Comment (armanio123) [inline Nodejs/Product/Nodejs/ProjectTemplates/AzureNodejsWorker/worker.js]: This looks like a code smell to me. It's ok for now as nobody has complain about it.

File: Nodejs/Product/Nodejs/ProjectTemplates/TypeScriptAzureExpressApp/app.ts [hunk 2]
```diff
@@ -28,7 +28,7 @@ app.use(function (req, res, next) {
 // development error handler
 // will print stacktrace
 if (app.get('env') === 'development') {
-    app.use((err: any, req, res, next) => {
+    app.use((err, req, res, next) => { // eslint-disable-line @typescript-eslint/no-unused-vars
```
Comment (armanio123) [inline Nodejs/Product/Nodejs/ProjectTemplates/TypeScriptAzureExpressApp/app.ts]: I would remove `next` instead. I don't know if the rule will still be required for `req`. I don't think so.

File: Nodejs/Product/Nodejs/ProjectTemplates/TypeScriptAzureExpressApp/app.ts [hunk 3]
```diff
@@ -39,7 +39,7 @@ if (app.get('env') === 'development') {
 
 // production error handler
 // no stacktraces leaked to user
-app.use((err: any, req, res, next) => {
+app.use((err, req, res, next) => { // eslint-disable-line @typescript-eslint/no-unused-vars
```
Comment (armanio123) [inline Nodejs/Product/Nodejs/ProjectTemplates/TypeScriptAzureExpressApp/app.ts]: `next` comment as before.

File: Nodejs/Product/Nodejs/ProjectTemplates/TypeScriptExpressApp/app.ts [hunk 4]
```diff
@@ -28,7 +28,7 @@ app.use(function (req, res, next) {
 // development error handler
 // will print stacktrace
 if (app.get('env') === 'development') {
-    app.use((err: any, req, res, next) => {
+    app.use((err, req, res, next) => { // eslint-disable-line @typescript-eslint/no-unused-vars
```
Comment (armanio123) [inline Nodejs/Product/Nodejs/ProjectTemplates/TypeScriptExpressApp/app.ts]: `next` comment as before.

File: Nodejs/Product/Nodejs/ProjectTemplates/TypeScriptExpressApp/app.ts [hunk 5]
```diff
@@ -39,7 +39,7 @@ if (app.get('env') === 'development') {
 
 // production error handler
 // no stacktraces leaked to user
-app.use((err: any, req, res, next) => {
+app.use((err, req, res, next) => { // eslint-disable-line @typescript-eslint/no-unused-vars
```
Comment (armanio123) [inline Nodejs/Product/Nodejs/ProjectTemplates/TypeScriptExpressApp/app.ts]: `next` comment as before.

File: Nodejs/Product/Nodejs/ProjectTemplates/TypeScriptAzureExpressApp/app.ts (hunk 2 above)
Comment (rachelgshaffer) [reply Nodejs/Product/Nodejs/ProjectTemplates/TypeScriptAzureExpressApp/app.ts]: "Error-handling middleware always takes four arguments. You must provide four arguments to identify it as an error-handling middleware function. Even if you don’t need to use the next object, you must specify it to maintain the signature." from [Express documentation](https://expressjs.com/en/api.html)

File: Nodejs/Product/Nodejs/ProjectTemplates/TypeScriptAzureExpressApp/app.ts (hunk 3 above)
Comment (rachelgshaffer) [reply Nodejs/Product/Nodejs/ProjectTemplates/TypeScriptAzureExpressApp/app.ts]: see other comment with link

File: Nodejs/Product/Nodejs/ProjectTemplates/TypeScriptExpressApp/app.ts (hunk 4 above)
Comment (rachelgshaffer) [reply Nodejs/Product/Nodejs/ProjectTemplates/TypeScriptExpressApp/app.ts]: see other comment with link

File: Nodejs/Product/Nodejs/ProjectTemplates/TypeScriptExpressApp/app.ts (hunk 5 above)
Comment (rachelgshaffer) [reply Nodejs/Product/Nodejs/ProjectTemplates/TypeScriptExpressApp/app.ts]: see other comment with link

File: Nodejs/Product/Nodejs/ProjectTemplates/AzureNodejsWorker/worker.js (hunk 1 above)
Comment (rachelgshaffer) [reply Nodejs/Product/Nodejs/ProjectTemplates/AzureNodejsWorker/worker.js]: fair enough

File: Nodejs/Product/Nodejs/ProjectTemplates/TypeScriptAzureExpressApp/app.ts (hunk 2 above)
Comment (armanio123) [reply Nodejs/Product/Nodejs/ProjectTemplates/TypeScriptAzureExpressApp/app.ts]: I found a bit weird the pattern but fair enough, disabling the eslint rule should do it. Good catch.

**Pull request merged**